userguide/plugins.rst

```diff
@@ -786,3 +787,16 @@ platform:
 * `DS-Client Management Guide <https://s3.amazonaws.com/asigra-documentation/Help/v14.1/DS-Client%20Help/index.html>`__:
   Managing the :literal:`DS-Client` system after it has been
   successfully installed at one or more locations.
+
+
+.. index:: community plugins, plugins
+.. _Community Plugins:
+
+Community Plugins
+-----------------
+
+%brand% offers the ability to switch plugin repositories to
+gain access to community created plugins. Switch the collection of
+plugins available by choosing a repository from the
```

Comment:
What are the repos available in the dropdown?  Should we tell them to choose "community-created"?  "Switch" is not clear here.  We should just tell them that there are different repos available and what they provide.